src/main/java/org/folio/circulation/infrastructure/storage/requests/RequestRepository.java

```diff
@@ -99,7 +99,8 @@ public static RequestRepository using(org.folio.circulation.support.Clients clie
 
     return findByWithoutItems(query, pageLimit)
       .thenComposeAsync(requests ->
-        itemRepository.fetchItemsFor(requests, Request::withItem));
+        itemRepository.fetchItemsFor(requests, Request::withItem))
+      .thenComposeAsync(multipleRecordsResult -> multipleRecordsResult.after(instanceRepository::findInstancesForRequests));
```

Comment:
What else will this affect except request queues?

Reply:
this change occured in findBy method of the repository and it is just invoked by RequestQueueRepository (getByInstanceId and getByItemId)